sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/IncrementalExecution.scala

```diff
@@ -77,7 +77,8 @@ class IncrementalExecution(
    */
   override
   lazy val optimizedPlan: LogicalPlan = tracker.measurePhase(QueryPlanningTracker.OPTIMIZATION) {
-    sparkSession.sessionState.optimizer.execute(withCachedData) transformAllExpressions {
+    sparkSession.sessionState.optimizer.executeAndTrack(withCachedData,
+      tracker) transformAllExpressions {
```

Comment:
nitpit: in most cases, we do not break the function call in the middle of parameter lists. We can change it to 
```Scala
    val sessionState = sparkSession.sessionState
    sessionState.optimizer.executeAndTrack(withCachedData, tracker).transformAllExpressions {
```
or 
```Scala
    sparkSession.sessionState.optimizer
      .executeAndTrack(withCachedData, tracker).transformAllExpressions {
```